templates/frontend/objects/article_details.tpl

```diff
@@ -120,7 +120,7 @@
 						{/if}
 						<div class="collapse" id="authorInfoCollapse">
 							{foreach from=$authors item=author key=number}
-								{if $author->getLocalizedAffiliation() || $author->getLocalizedBiography()}
+								{if $author->getLocalizedAffiliationNamesAsString() || $author->getLocalizedBiography()}
```

Comment:
Maybe to use `count($author->getAffiliations()) > 0` instead of `$author->getLocalizedAffiliationNamesAsString()` -- getLocalizedAffiliationNamesAsString() does more processing... and it seems it is not used...